### ISSUE TITLE
Segmentation fault with "system"

### ISSUE THREAD

**Issue by ghost** [ThomasDickey/original-mawk#51]
~~~sh
$ mawk -W version
mawk 1.3.4 20171017

$ mawk 'BEGIN {system("echo")}'
Segmentation fault
~~~

Note this is on Windows version, Linux version works as expected.

**Comment by svnpenn** [comment]
https://github.com/svnpenn/velour/issues/100

**Comment by ThomasDickey** [comment]
This might depend on locale or specific configure options (I don't see it with a default configure and en_US.UTF-8).

**Comment by ThomasDickey** [comment]
fixed in [20230804](https://invisible-island.net/mawk/CHANGES.html#t20230804):
>add MinGW-code for system() calls
